environment/src/main/java/com/sequenceiq/environment/environment/flow/deletion/handler/ResourceEncryptionDeleteHandler.java

```diff
@@ -42,7 +74,10 @@ public void accept(Event<EnvironmentDeletionDto> environmentDtoEvent) {
         EnvironmentDto environmentDto = environmentDeletionDto.getEnvironmentDto();
         EnvDeleteEvent envDeleteEvent = getEnvDeleteEvent(environmentDeletionDto);
         try {
-            environmentService.findEnvironmentById(environmentDto.getId()).ifPresent(this::deleteEncryptionResources);
+            environmentService.findEnvironmentById(environmentDto.getId()).ifPresent(environment -> {
+                String diskEncryptionSetUrl = environment.getEnvironmentEncryptionParameters();
```

Comment:
It is a Resource ID, not an URL. But its placement is wrong anyway as mentioned above.